src/tilemap_layer.cpp

```diff
@@ -681,13 +685,6 @@ std::vector<unsigned char> TilemapLayer::GetPassable() const {
 void TilemapLayer::SetPassable(const std::vector<unsigned char>& npassable) {
 	passable = npassable;
 
-	if (substitutions.size() < passable.size())
-	{
-		substitutions.resize(passable.size());
-		for (uint8_t i = 0; i < substitutions.size(); i++)
-			substitutions[i] = i;
-	}
```

Comment:
Was this just here for extra safety or is there some specific case its needed? Both these arrays should always be size 144.

Reply:
hmmm, fcdf1b1be548.

Reply:
can't comment on this, the chatlogs of February 2012 aren't useful either

Reply:
don't want to link our chatlog publicly but take a look at 2011/2011-01-16.
has some substitution discussion, though there everybody says it is 144 because the autotiles are ignored.

Reply:
The LSD data from RPG_RT is always size 144.